package.json

```diff
@@ -59,6 +60,11 @@
         "command": "memo.openReferenceInDefaultApp",
         "title": "Open link in the default app",
         "category": "Memo"
+      },
+      {
+        "command": "memo.openReferenceInNextColumn",
+        "title": "Open link in the next column of this editor",
```

Comment:
Thanks. that makes sense.  @svsool  What would be your thought on "Open link to the side", or "Open to the side" same as the existing context menu in Explorer file tree? They almost have the same behavior, instead of opening a file corresponding to the tree node, we are opening a link referenced file to the side. Just a thought...
![image](https://user-images.githubusercontent.com/21694444/104853307-cbb61f00-58b4-11eb-8a0f-13737f987bf9.png)

We could bind to the same hotkey (ctrl+enter).

Reply:
Sounds good 👍 .

Reply:
just fyi, later on I noticed that the hotkey <kbd>ctrl</kbd>+<kbd>enter<kbd> has already been bound to a different cmd "editor.action.openLink".  so I changed my mind on that.